runtime/moonbase/src/lib.rs

```diff
@@ -1330,6 +1330,7 @@ pub type SignedExtra = (
 	frame_system::CheckNonce<Runtime>,
 	frame_system::CheckWeight<Runtime>,
 	pallet_transaction_payment::ChargeTransactionPayment<Runtime>,
+	frame_system::CheckNonZeroSender<Runtime>,
```

Comment:
The `CheckNonZeroSender` type should be added at the beginning of the tuple, not the end (like poladot does, it's because we should start with the fastest checks, those that not require any DB read)

Reply:
Changed